lib/presentation/login_page/login_page.dart

```diff
@@ -1,11 +1,20 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:form_field_validator/form_field_validator.dart';
 import 'package:get/get.dart';
 import 'package:lottie/lottie.dart';
+import 'package:weather/cubit/sign_in/sign_in_cubit.dart';
+import 'package:weather/cubit/sign_up/sign_up_cubit.dart';
+import 'package:weather/cubit/user/user_cubit.dart';
 import 'package:weather/presentation/login_page/components/email_form_field.dart';
 import 'package:weather/presentation/login_page/components/login_button.dart';
 import 'package:weather/presentation/login_page/components/password_form_field.dart';
 import 'package:weather/presentation/login_page/components/register_button.dart';
+import 'package:weather/utils/authentications.dart';
+import 'package:weather/utils/autocomplete_show_dialog.dart';
+import 'package:weather/utils/error_dialog.dart';
+import 'package:firebase_auth/firebase_auth.dart' as fbAuth;
```

Comment:
IDE wants here fb_Auth, don't know why, but that's a warning so you can change it